app/presenters/search_presenter.rb

```diff
@@ -11,7 +11,11 @@ def to_json
       description: document.summary,
       link: document.base_path,
       indexable_content: indexable_content,
-      organisations: organisation_slugs,
+      # FIXME: This line was dependent on a method called #organisation_slugs,
+      # which in turn used publishing_api.get_content to map org content_ids to
+      # organisation slugs. It was stubbed in many places in spec/, but that method
+      # call always returned a blank list in the real world.
+      organisations: [],
```

Comment:
Since we're working off the assumption that [Rummager will index the organisations from Publishing API](https://trello.com/c/RrN7B6xQ/593-rework-the-way-rummager-indexes-links) very soon, do we need the FIXME and to send the organisations array at all? We can always revert this PR if the assumption turns out to be incorrect for whatever reason.